packages/sdk-codegen/src/sdkModels.ts

```diff
@@ -1707,18 +1740,30 @@ export class ApiModel implements ISymbolTable, IApiModel {
     return result
   }
 
-  // if any properties of the parameter type are readOnly (including in subtypes)
-
-  sortedTypes() {
-    return Object.values(this.types).sort((a, b) =>
-      a.name.localeCompare(b.name)
-    )
+  /**
+   * Sort a keyed collection so its keys are in sorted order
+   * @param {IKeyedCollection<T>} list to sort
+   * @returns {IKeyedCollection<T>} newly sorted list
+   */
+  sortList<T>(list: IKeyedCollection<T>): IKeyedCollection<T> {
+    const result = {}
+    const sortedKeys = Object.keys(list).sort((a, b) => a.localeCompare(b))
+    for (const key of sortedKeys) {
+      result[key] = list[key]
+    }
+    return result
```

Comment:
nice!

Reply:
I'd prefer a sorted dictionary type/interface but this will have to do for now